conda_smithy/templates/run_win_build.bat.tmpl

```diff
@@ -56,11 +56,18 @@ if EXIST LICENSE.txt (
     copy LICENSE.txt "{{ recipe_dir }}\\recipe-scripts-license.txt"
 )
 
-{%- if test in ["native", "native_and_emulated"] %}
+{%- if test == "native" %}
 if NOT [%HOST_PLATFORM%] == [%BUILD_PLATFORM%] (
     set "EXTRA_CB_OPTIONS=%EXTRA_CB_OPTIONS% --no-test"
 )
 {%- endif %}
+{%- if test == "native_and_emulated" %}
+if NOT [%HOST_PLATFORM%] == [%BUILD_PLATFORM%] (
+    if NOT [%CROSSCOMPILING_EMULATOR%] == [1] (
```

Comment:
I mentionned in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/300#issuecomment-1826928079 but I'm going to repeat the comment here:

On linux `CROSSCOMPILING_EMULATOR` is a path. On windows it is a boolean. I feel like it will be a problem later on.

Reply:
Can we update the test to be:
```
if NOT [%CROSSCOMPILING_EMULATOR%] == [] 
```

in the case to give a prallel to the linux case, where non-emptyness means "valid"